test/OpenTelemetry.Tests/Impl/Metrics/TestMetricProcessor.cs

```diff
@@ -27,6 +27,14 @@ internal class TestMetricProcessor : MetricProcessor
         public ConcurrentBag<Metric<long>> longMetrics = new ConcurrentBag<Metric<long>>();
         public ConcurrentBag<Metric<double>> doubleMetrics = new ConcurrentBag<Metric<double>>();
 
+        public override Tuple<IEnumerable<Metric<long>>, IEnumerable<Metric<double>>> FinishCollectionCycle()
```

Comment:
Curious why return a tuple, not out parameters? Using of this method looks strange with all the `Item1` properties of a Tuple. Also more extendable in future.

Reply:
changed to out param.
Once the generics from instruments are removed, I will be modifying the exportrecord (`MetricData`) so that it'll be just single `List<MetricData>` in the future.